alfresco-transform-libreoffice/alfresco-transform-libreoffice-boot/src/main/resources/application-default.yaml

```diff
@@ -5,4 +5,4 @@ transform:
     config:
       location: classpath:libreoffice_engine_config.json
     libreoffice:
-      home: ${LIBREOFFICE_HOME:/opt/libreoffice6.3}
```

Comment:
Should this be changing?

Reply:
Yes, both the AIO controller and the LibreOffice controller application.yml file were given different variable names, which prevented inheritance of the test class.